.github/workflows/mc-ledger-test.yml

```diff
@@ -0,0 +1,98 @@
+name: mc-ledger-test
+
+env:
+  SGX_MODE: SW
+  IAS_MODE: DEV
+  RUST_BACKTRACE: full
+  IAS_API_KEY: ${{ secrets.SGX_PRIMARY_API_KEY }}
+  IAS_SPID: ${{ secrets.SGX_SPID }}
+  MC_LOG: warn
+
+on:
+  workflow_dispatch:
+    inputs:
+      version_a_ref:
+        description: 'Tag or commit-hash for the base version of Full-Service'
+        type: string
+        required: true
+        default: 'v1.9.0' 
+      version_b_ref:
+        description: 'Tag or commit-hash to upgrade Full-Service to'
+        type: string
+        required: true
+        default: "v1.9.2"
+  push:
+    branches:
+      - setup-mobilecoin
+      - pouneh/e2e_test_infra
+    inputs: 
+      version_a_ref:
+        description: 'Tag or commit-hash for the base version of Full-Service'
+        required: true
+        default: 'v1.10.0-pre.5' #'v1.9.0' 
```

Comment:
Why is this version newer than the one being upgraded to? Where are these versions coming from and when is someone expected to change them?